source/_integrations/zha.markdown

```diff
@@ -241,46 +241,32 @@ custom_quirks_path:
   type: string
 {% endconfiguration %}
 
-### OTA firmware updates
+### OTA updates of Zigbee device firmware
 
-The ZHA {% term integration %} has the ability to automatically download and perform OTA (Over-The-Air) firmware updates of Zigbee devices if the OTA firmware provider source URL for updates is available. OTA firmware updating is set to disabled (`false`) in the configuration by default.
+The ZHA integration have the ability to perform firmware updates of some specific Zigbee devices if they both support the standard Zigbee OTAU (Over-The-Air Update) feature and firmware images with standard Zigbee OTA format that brand and device are provided to the public by the manufacturer. You will get notifications if and when a device has firmware update available and to start an upgrade you have to initiate it by click a button.
 
-Online OTA providers for firmware updates are currently only available for IKEA, LEDVANCE/OSRAM, SALUS/Computime, and INOVELLI devices. Support for OTA updates from other manufacturers could be supported in the future if they publish their firmware images publicly.
+OTA providers from a few manufacturer's official online sources that allow firmware update images are currently only available for Zigbee device brands from; IKEA, INOVELLI, LEDVANCE (OSRAM), SALUS (Computime), Sonoff (ITead), and 3reality (Third Reality). Automatic OTA firmware updating is set to disabled in the configuration by default, and if you set it to enabled for a brand then ZHA will currently automatically update firmware without notifications on any compatible device as soon as it finds a compatible image available on local storage.
```

Comment:
Unclear for me at least: how to enable automatic updates? Same provider tags described below to disable provider updates? Or something else? 
If they are the same options, it's a little unexpected. Because the config is a tri state then: (true, undefined, false) ?!

Reply:
This isn't correct, there is no way to enable automatic updates. ZHA uses standard Home Assistant update entities and it's no different from any other integration in this regard.

Reply:
Please re-review latest commit with rewrite https://github.com/home-assistant/home-assistant.io/pull/28895/commits/50f9fe399bb5f94701f56a6bb857f6e23fde3603